cmd/skopeo/copy.go

```diff
@@ -54,7 +54,7 @@ func copyHandler(context *cli.Context) {
 	}
 	signBy := context.String("sign-by")
 
-	manifest, err := src.GetManifest()
+	manifest, _, err := src.GetManifest()
```

Comment:
(This is the right thing to do now.

Eventually we might need to do conversions, i.e. turn the core of `copy` into a function available in the library (which would be called for `pull` operation by k8s).  But that’s for much later, if ever.)